docs/src/examples/QasTableGenerator/WithFieldsProps.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <qas-table-generator :fields :results v-bind="tableGeneratorProps" />
```

Comment:
pq usou o v-bind pra passar apenas o tableGeneratorProps? se n for centralizar as props em um unico v-bind, coloca `:table-generator-props`

Reply:
Centralizei td no tableGeneratorProps mas não entendi o `:table-generator-props`